_data/sidebar_doc.yml

```diff
@@ -4,249 +4,92 @@
 entries:
 - title: Sidebar
   subcategories:
-    - title: Frontmatter # leave this frontmatter section here for PDF outputs
-      audience: all
-      platform: all
-      product: all
-      version: all
-      web: false
-      items:
-        - title: Title Page
-          url: /titlepage.html
-          audience: all
-          platform: all
-          product: all
-          version: all
-          web: false
-
-        - title: Table of Contents
-          url: /tocpage.html
-          audience: all
-          platform: all
-          product: all
-          version: all
-          web: false
-
     - title: Get Started
-      audience: all
```

Comment:
These tie into the theme's ability to tag content for specific outputs. So you can have one `md` file, for example, with conditional logic that makes it suitable for multiple outputs (paragraph 1 for version 1, paragraph 2 for version 2, etc.). This type of thing reduces duplication (i.e., unique copy of same file per version), which makes updating docs much more efficient. Long-term (and I know this isn't even remotely relevant for Cockroach), it dramatically reduces translation costs as well. 

Anyway, we're not using this now, so it's fine to remove. Once you get to thinking about how to handle versions, though, I'd like to talk it through and make sure our approach is as future-oriented as possible.